docs/sources/tempo/traceql/metrics-queries/solve-problems-metrics-queries.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Solve problems with trace metrics queries
+menuTitle: Use cases
+description: Solve problems with trace metrics queries
+weight: 600
+keywords:
+  - metrics query
+  - TraceQL metrics
+---
+
+# Solve problems with trace metrics queries
+
+Or, TraceQL metrics for the PromQL user
+
+You can query data generated by TraceQL metrics in a similar way that you would query results stored in Mimir.
+TraceQL metrics provides calculated metrics on trace span data, without requiring you to utilize an telemetry signal.
+
+This page provides an example of how you can investigate the rate of incoming requests using both PromQL and TraceQL.
+
+## RED metrics and queries
+
+The Tempo metrics-generator emits metrics with pre-configured labels for Rate, Error, and Duration (RED) metrics and service graph edges.
+Generated metric labels vary, but always include the service name (in service graph metrics, as a client and/or a server type).
+For more information, refer to the [metrics-generator documentation](../../metrics-generator/).
+
+You can use these metrics to get an overview of application performance.
+The metrics can be directly correlated to the trace spans that are available for querying.
+
+TraceQL metrics provide functionality that lets you make a variety of requests directly to Tempo instead of requiring the metrics-generator component.
```

Comment:
```suggestion
TraceQL metrics allow a user to query metrics from traces directly from Tempo instead of requiring the metrics-generator component and an accompanying TSDB.
```